setup.cfg

```
@@ -28,33 +29,50 @@ classifiers=
 python_requires = >=3.6
 packages = find:
 include_package_data = True
-setup_requires = setuptools_scm
+setup_requires =
+  # PEP-518 encourages the use of pyproject.toml for defining
+  # build dependencies...they should be defined under build-system.requires
 install_requires =
+  # ought to mirror requirements/install.txt
   astropy >= 3.1
   colorama >= 0.3
+  matplotlib
```

```suggestion
  matplotlib >= 2.0
```
We ought to specify a minimum version here too, though I'm pretty much guessing with 2.0.

---

I think it should be `matplotlib >= 3.2`.  That's when they started supporting python 3.8 (3.6+).  `2.0` still supports python 2.

---

`matplotlib >= 3.2` sounds good to me!